### ISSUE TITLE
[Discussion] Stems and 16khz Frequency usage

### ISSUE THREAD

**Issue by deskstar90** [deezer/spleeter#272]
I'm rather new at Spleeter, about 10 songs split as of today. I read that it can also separate at 16khz. When I tried this to see the difference, ...-c audio.mp3 spleeter:4stems-16khz -o output
I got the following msg; error: spleeter: no embedded cfs(I think it was) 4stems-16khz found
Is this still functional or is there to know what would be the optimal freq to use? Sometimes I get 2 tracks or 3 split tracks but no sounds in either one. Is this a bug or perhaps just because it cannot split it for some reason?

**Comment by boringhexi** [comment]
Not sure if this is your cause, but I had a similar thing happen when I forgot to capitalize the H in `-p spleeter:4stems-16kHz`

**Comment by deskstar90** [comment]
no, that is not the issue. Let me ask you, how do you run the command..
i.e -p spleeter:4stems-16khz -o output OR -c spleeter:4stems-16khz -o output OR other..?
and I imagine the 16Khz files are in the pretrained_models folder? 
thanks to anyone who can shed some light..

**Comment by deskstar90** [comment]
ok, I figured it out but using "-p spleeter:4stems" or "-p spleeter:4stems-16kHz" still downloads the same pretrained model file "4stems.tar.gz" there is a finetune version but running -p spleeter:4stems-finetune did not download the proper pretrained model.  Should it work If I  just copy them into the pretrained_models folder?.. I'll try it out

**Comment by martobg10** [comment]
> ok, I figured it out but using "-p spleeter:4stems" or "-p spleeter:4stems-16kHz" still downloads the same pretrained model file "4stems.tar.gz" there is a finetune version but running -p spleeter:4stems-finetune did not download the proper pretrained model. Should it work If I just copy them into the pretrained_models folder?.. I'll try it out

What's the result? Do you find a solution?

**Comment by deskstar90** [comment]
I did not see or hear any difference in using either files if anyone cares to know. Also using different files sources, i.e mp3, wav, ogg, flac did not make any difference either(or that I can hear). but I think those artefacts are partly due to the mastered original including fx being split into stems compared to dry multitracks.

**Comment by MrBasssman65** [comment]
check issue # 233 for my workaround solution to only use 16kZHz files.

**Comment by deskstar90** [comment]
MrBasssman65, I had already found your tip and followed it and did get the 16kHz files loaded. Except, I cannot tell the difference between the two soundwise. The difference might be very minimal, like comparing an Mp3 to a lossless WAV file, due to the higher frequencies being trimmed. I'm not sure there is much more that can be done about the artefacts. one thing I noticed, the bass tracks are very muddy, It doesn't do a good job on the lower frequencies, I have to run them thru a plugin to get a better sounding bass.

**Comment by Smopraq** [comment]
I think you need to train the AI with the genre you are wanting to split. It just hasn't learned how to do it yet.

**Comment by deskstar90** [comment]
I'm willing to try training but I'm not sure how to go about it, is there a good read on it somewhere to get me started?
